src/theme/semantic-tokens/colors.ts

```diff
@@ -7,6 +7,7 @@ export const colors = {
   "brand.100": "#FBE4D7",
   "brand.800": "#D96B27",
   "brand.900": "#C85124",
+  "gray.700": "#2D3748",
```

Comment:
You should be able to delete this and have your styling still work. If you reference "gray.700", Chakra should use the gray.600 value in `/tokens/colors.ts`.

I realize my instructions on when to use semantic versus regular tokens were a little unclear - It's ok to use tokens in the "regular' tokens file when there isn't a semantic token pointing to the underlying regular token or "raw" value. Because we don't have a semantic name for this gray, you can just reference the regular token for it.